DevEdu.API/DevEdu.Business.Tests/LessonServiceTests.cs

```diff
@@ -0,0 +1,185 @@
+﻿using DevEdu.Business.Services;
+using DevEdu.Business.ValidationHelpers;
+using DevEdu.DAL.Models;
+using DevEdu.DAL.Repositories;
+using Moq;
+using NUnit.Framework;
+
+namespace DevEdu.Business.Tests
+{
+    public class LessonServiceTests
+    {
+        private Mock<ILessonRepository> _lessonRepoMock;
+        private Mock<ICommentRepository> _commentRepoMock;
+        private Mock<IUserRepository> _userRepoMock;
+        private Mock<IUserValidationHelper> _userValidationHelperMock;
+        private Mock<ILessonValidationHelper> _lessonValidationHelperMock;
+
+
+        [SetUp]
+        public void Setup()
+        {
+            _lessonRepoMock = new Mock<ILessonRepository>();
+            _commentRepoMock = new Mock<ICommentRepository>();
+            _userRepoMock = new Mock<IUserRepository>();
+            _userValidationHelperMock = new Mock<IUserValidationHelper>();
+            _lessonValidationHelperMock = new Mock<ILessonValidationHelper>();
+        }
+
+        [Test]
+        public void AddStudentToLesson_IntLessonIdAndUserId_AddingStudentToLesson()
+        {
+            //Given
+            var studentLessonDto = LessonData.GetStudentLessonDto();
+            
+            var lessonId = LessonData.LessonId;
+            var userId = LessonData.UserId;
+
+            _lessonRepoMock.Setup(x => x.AddStudentToLesson(lessonId, userId));
+            _lessonRepoMock.Setup(x => x.SelectByLessonAndUserId(lessonId, userId)).Returns(studentLessonDto);
```

Comment:
bad naming: "SelectByLessonAndUserId"
please rename to SelectAttendanceByLessonAndUserId